doc/source/example_notebooks/IS2_data_access.ipynb

```diff
@@ -83,7 +83,7 @@
     "*NOTE: The input keyword for `short_name` was updated in the code from `dataset` to `product` to match common usage.\n",
     "This should not affect users providing positional inputs as demonstrated in this tutorial.*\n",
     "\n",
-    "Then, you must include AT LEAST one of the following inputs (temporal or orbital constraints):\n",
+    "Then, for all non-gridded products (ATL<=12), you must include AT LEAST one of the following inputs (temporal or orbital constraints):\n",
```

Comment:
```suggestion
    "Then, for all non-gridded products (ATL<=13), you must include AT LEAST one of the following inputs (temporal or orbital constraints):\n",
```

Reply:
I have this fix committed locally and forgot to push! Thanks for catching it again!